### DISCUSSION TITLE
openapi返回错误

### DISCUSSION THREAD

**Discussion by 2478641181** [whyour/qinglong#1621] [Q&A]
{"statusCode":400,"error":"Bad Request","message":"Validation failed","validation":{"body":{"source":"body","keys":[""],"message":"\"value\" must be an array"}}}
求解,为什么网页版提交信息不通用 那么应该提交什么？
提交如下
[{"name":"qqq12321","value":[1,2]","remarks":"123"}]

**Comment by whyour** [comment]
> {"statusCode":400,"error":"Bad Request","message":"Validation failed","validation":{"body":{"source":"body","keys":[""],"message":"\"value\" must be an array"}}}
> 
> 求解,为什么网页版提交信息不通用 那么应该提交什么？
> 
> 提交如下
> 
> [{"name":"qqq12321","value":[1,2]","remarks":"123"}]

哪个api  什么叫不通用
